Makefile

```diff
@@ -17,7 +17,12 @@ export GO15VENDOREXPERIMENT=1
 
 # Bump these on release
 VERSION ?= v0.9.0
+# There's definitely a better way to do this, should I use cut from the VERSION?
+# Or should we have major, minor, and build and then construct VERSION/DEB_VERSION from that?
```

Comment:
Yeah I like breaking it into major/minor/build and then constructing these from that.